.idea/phpstorm-stubs.iml

```diff
@@ -2,6 +2,7 @@
 <module type="WEB_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/spec" isTestSource="true" />
```

Comment:
Could you please remove this from the .iml file & commit again? That's an incorrect line added because of a bug: https://youtrack.jetbrains.com/issue/WI-57452

Reply:
Ohh right, sorry.